src/types/state.js

```diff
@@ -318,6 +351,7 @@ export type ProfileSpecificUrlState = {|
   transforms: TransformStacksPerThread,
   timelineType: TimelineType,
   sourceView: SourceViewState,
+  assemblyView: AssemblyViewState,
```

Comment:
I see that his is not (un/)serialized in the `url-handling` code, do you intend to serialize this data later? If not, it might make sense to move it to another state, maybe `AppState`?

Reply:
I do intend to serialize this data later. I have the patch (it's in the full series in the other PR) but I wanted to write tests for it first.